.env

```diff
@@ -16,6 +16,7 @@ IDAM_KEY_CCD_GATEWAY=AAAAAAAAAAAAAAAC
 IDAM_KEY_CCD_ADMIN=AAAAAAAAAAAAAAAD
 IDAM_KEY_DM_STORE=AAAAAAAAAAAAAAAA
 IDAM_KEY_FPL_CASE_SERVICE=AABBCCDDEEFFGGHH
+IDAM_KEY_XUI_WEBAPP=AAAAAAAAAAAAAAAF
```

Comment:
can you please add al XUI specific env variables in this file and the others into a distinct section `#  XUI`

Reply:
done